### PR TITLE
Add about modal

### PR DESCRIPTION
**Pull request by guillaumevincent**
Add a About link button in the header to display a modal with version
number

https://github.com/ansible-community/ara/issues/248

### CODE REVIEW

**Comment by guillaumevincent** [comment]
image

**Comment by guillaumevincent** [comment]
image

File: ara/ui/templates/partials/modal_about.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          type="button"
+          class="close"
+          data-dismiss="modal"
+          aria-label="Close"
+        >
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <ul class="list-group list-group-flush">
+        <li class="list-group-item">Created by David Moreau Simard</li>
```
Comment (dmsimard) [inline ara/ui/templates/partials/modal_about.html:23]: Thank you but unnecessary :)
Let's leave that line out.

**Comment by kfiresmith** [comment]
Thanks for providing this feature - appreciate it!

File: ara/server/context_processors.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021 The ARA Records Ansible authors
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+import pbr.version
+
+from django.conf import settings
```
Comment (dmsimard) [inline ara/server/context_processors.py:6]: linter unhappy about this one

File: ara/ui/templates/partials/modal_about.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  id="aboutModal"
+  tabindex="-1"
+  role="dialog"
+  aria-labelledby="aboutModalLabel"
+  aria-hidden="true"
+>
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="aboutModalLabel">Ara {{ ARA_VERSION }}</h5>
```
Comment (dmsimard) [inline ara/ui/templates/partials/modal_about.html:12]:
```suggestion
        <h5 class="modal-title" id="aboutModalLabel">ara {{ ARA_VERSION }}</h5>
```

**Comment by dmsimard** [comment]
Since the version is already in the modal header, maybe we don't need to repeat it in the modal body.

Now that there would be no content, maybe we should add stuff like:
- Website: https://ara.recordsansible.org
- Code: https://github.com/ansible-community/ara
- License: GPLv3

**Comment by dmsimard** [comment]
@guillaumevincent I'll take care of this tomorrow.

**Comment by dmsimard** [comment]
Going to close this one and merge #257 instead. Thanks !

**Pull request closed without merging**